cibuildwheel/windows.py

```diff
@@ -519,6 +524,10 @@ def build(options: Options, tmp_path: Path) -> None:
                 # check that we are using the Python from the virtual environment
                 call("where", "python", env=virtualenv_env)
 
+                # TODO remove me once virtualenv provides pip>=24.1b1
+                if config.version.startswith("3.13."):
+                    call("python", "-m", "pip", "install", "--pre", "-U", "pip", env=virtualenv_env)
```

Comment:
This does not work with `call("python", "-m", "pip", "install", "pip>=24.1b1", env=virtualenv_env)`
Maybe something to do with `shell=True` being used on Windows ?